### DISCUSSION TITLE
First release

### DISCUSSION THREAD

**Discussion by hawkeye116477** [hawkeye116477/playnite-legendary-plugin#5] [Announcements]
* Bugs 🙂

<hr /><em>This discussion was created from the release <a href='https://github.com/hawkeye116477/playnite-legendary-plugin/releases/tag/1.0.0'>First release</a>.</em>

**Comment by fruselight** [comment]
Will be there any support for Rare launcher since it's basically GUI front-end of Legendary?

**Reply by hawkeye116477** [reply]
No, only Legendary will be supported and I can't also reuse Legendary from Rare, cuz it's included the other way than it's in Heroic.
